plugins/speculation-rules/class-plsr-url-pattern-prefixer.php

```diff
@@ -107,13 +107,13 @@ public function prefix_path_pattern( string $path_pattern, string $context = 'ho
 	 */
 	public static function get_default_contexts(): array {
 		return array(
-			'home'       => self::escape_pattern_string( trailingslashit( wp_parse_url( home_url( '/' ), PHP_URL_PATH ) ) ),
-			'site'       => self::escape_pattern_string( trailingslashit( wp_parse_url( site_url( '/' ), PHP_URL_PATH ) ) ),
-			'uploads'    => self::escape_pattern_string( trailingslashit( wp_parse_url( wp_upload_dir( null, false )['baseurl'], PHP_URL_PATH ) ) ),
-			'content'    => self::escape_pattern_string( trailingslashit( wp_parse_url( content_url(), PHP_URL_PATH ) ) ),
-			'plugins'    => self::escape_pattern_string( trailingslashit( wp_parse_url( plugins_url(), PHP_URL_PATH ) ) ),
-			'template'   => self::escape_pattern_string( trailingslashit( wp_parse_url( get_stylesheet_directory_uri(), PHP_URL_PATH ) ) ),
-			'stylesheet' => self::escape_pattern_string( trailingslashit( wp_parse_url( get_template_directory_uri(), PHP_URL_PATH ) ) ),
+			'home'       => self::escape_pattern_string( trailingslashit( (string) wp_parse_url( home_url( '/' ), PHP_URL_PATH ) ) ),
```

Comment:
Hmm I thought `szepeviktor/phpstan-wordpress` handled the `wp_parse_url` return type

Reply:
Humm. Yeah, you're right: https://github.com/szepeviktor/phpstan-wordpress/blob/74b9e2f572b51cab33259b3c55a88050d3cbe28b/src/WpParseUrlFunctionDynamicReturnTypeExtension.php

Reply:
Ah, this is the error:

> Parameter #1 $value of function trailingslashit expects string, string|false|null given.

So the issue isn't that it thinks an array is being returned, but rather an error condition is being returned. So I think this is correct.